canonical-web-and-design/snapcraft.io
# Release UI branches

Fixes https://github.com/canonical-web-and-design/snapcraft.io/issues/2076
Fixes https://github.com/canonical-web-and-design/snapcraft.io/issues/2079
Fixes https://github.com/canonical-web-and-design/snapcraft.io/issues/2081

## Note
You'll need a snap with branches. The easiest way to open a branch, that I've found, is to release a snap into a branch: `snapcraft release <snap_name> <revision_number> <branch (e.g. latest/stable/test)>`.

## QA
- Pull the branch
- `./run`
- Visit http://0.0.0.0:8004/<snap_name>/releases
- Release revisions to branches
- Close revisions in branches
- Promote to and from branches
- Drag things around
- Check the history of branch architectures
- Generally just click and drag things around as a normal user would
